src/main/rx/expresslrs_common.c

```diff
@@ -262,6 +272,12 @@ uint16_t txPowerIndexToValue(const uint8_t index)
     }
 }
 
+uint32_t elrsUidToSeed(const uint8_t UID[])
+{
+    return ((uint32_t)UID[2] << 24) + ((uint32_t)UID[3] << 16) +
+        ((uint32_t)UID[4] << 8) + (UID[5] ^ ELRS_OTA_VERSION_ID);
+}
```

Comment:
```suggestion
    return ((uint32_t)UID[2] << 24) | ((uint32_t)UID[3] << 16) |
        ((uint32_t)UID[4] << 8) | (UID[5] ^ ELRS_OTA_VERSION_ID);
}
```

Also, is it guaranteed that ELRS_OTA_VERSION_ID < 256 ?